components/tools/OmeroWeb/test/integration/test_thumbnails.py

```diff
@@ -36,6 +41,18 @@
 standard_library.install_aliases()
 
 
+def rgba_to_int(red, green, blue, alpha=255):
```

Comment:
such method already exists in ``test_api_rois.py``, it will make more sense to move it to a common location.